frontend/packages/kubevirt-plugin/integration-tests-cypress/tests/template/template-resource.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { testName } from '../../support';
+import { Disk, Network } from '../../types/vm';
+import { TEMPLATE } from '../../utils/const/index';
+import { addDisk, addNIC, deleteRow } from '../../views/dialog';
+import { tab } from '../../views/tab';
+import { virtualization } from '../../views/virtualization';
+
+const nic1: Network = {
+  name: 'nic-1',
+  nad: 'bridge-network',
+};
+
+const disk1: Disk = {
+  name: 'disk-1',
+  size: '1',
+};
+
+describe('Test Template resource', () => {
```

Comment:
nitpick: `Template` shouldn't be capitalized here.

Reply:
NP